src/net.cpp

```diff
@@ -112,9 +112,9 @@ static const uint64_t RANDOMIZER_ID_ADDRCACHE = 0x1cf2e4ddd306dda9ULL; // SHA256
 //
 bool fDiscover = true;
 bool fListen = true;
-RecursiveMutex cs_mapLocalHost;
-std::map<CNetAddr, LocalServiceInfo> mapLocalHost GUARDED_BY(cs_mapLocalHost);
-static bool vfLimited[NET_MAX] GUARDED_BY(cs_mapLocalHost) = {};
+Mutex map_localhost_mutex;
```

Comment:
Suggesting to add `g_` prefix to explicitly note that this mutex belongs to the global namespace:
```suggestion
Mutex g_maplocalhost_mutex;
```

Also see #19180.

Reply:
Thanks for suggestion. Done.